src/main/java/reposense/util/FileUtil.java

```diff
@@ -57,55 +65,77 @@ public static void deleteDirectory(String root) throws IOException {
         }
     }
 
-    public static void unzip(ZipInputStream zipInput, String destinationFolder) {
-        Path directory = Paths.get(destinationFolder);
-
-        // buffer for read and write data to file
-        byte[] buffer = new byte[2048];
+    /**
+     * Zips all the files of {@code fileType} contained in the {@code sourceAndOutputPath}.
+     * Zipped file is contained in the same location {@code sourceAndOutputPath} as {@code fileTypes}'s location.
+     */
+    public static void zip(Path sourceAndOutputPath, String... fileType) {
```

Comment:
`Zips all the files of {@code fileType} contained inside {@code sourceAndOutputPath} directory into the same folder.`?